docs/anonymous-statistics.md

```diff
@@ -24,7 +24,7 @@ You can opt-out from sending anonymous statistics to Netdata through three diffe
 
 ## Agent Dashboard - PostHog JavaScript
 
-When you kick off an Agent dashboard session by visiting `http://NODE:19999`, Netdata will initialiszes a PostHog session and masks various event attributes.
+When you kick off an Agent dashboard session by visiting `http://NODE:19999`, Netdata will initializes a PostHog session and masks various event attributes.
```

Comment:
Not your fault (:smiley: ), but this should be `initialize`, no `s`.

Or, we can remove `will`. I actually prefer the latter, but don't want to make your life any harder!

Reply:
Fixing a line isn't a big deal (especially since this doesn't change the commit message!). I'll wait for the other thing below before I do it.

Reply:
Let's fix this

Reply:
```suggestion
When you kick off an Agent dashboard session by visiting `http://NODE:19999`, Netdata will initialize a PostHog session and masks various event attributes.
```